<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student</class>
 <widget class="QDialog" name="Student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>627</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>学生借阅系统</string>
  </property>
  <widget class="QPushButton" name="book_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>书名查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="user_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>490</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>个人管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="borrow_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>借阅情况</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ISBN_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>ISBN查询</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>831</width>
     <height>601</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>书名：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_search_text">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="name_search_button">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="name_search_table">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>631</width>
       <height>341</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>ISBN</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>书名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>作者</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>出版社</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>在馆</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="all_books_button">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>40</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>全部图书</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ISBN:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ISBN_search_text">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ISBN_search_button">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>80</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="ISBN_decription">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>441</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>72</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>简介：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>150</y>
       <width>72</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>书  名：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ISBN_book_name">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>150</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>250</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>作  者：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>330</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>出版社：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>420</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>在  馆：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ISBN_author">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>240</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ISBN_publisher">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>320</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ISBN_onlibrary">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>410</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>页</string>
    </attribute>
    <widget class="QTableWidget" name="borrow_book_table">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>761</width>
       <height>231</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>ISBN</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>书名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>作者</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>出版社</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>开始时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>结束时间</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>72</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ISBN:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="borrow_ISBN_text">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="borrow_ISBN_button">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>120</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>借阅</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back_ISBN_button">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>120</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>归还</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>您的借阅书目：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>借阅时间：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="day_combox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>页</string>
    </attribute>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>72</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>学号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="student_sno">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>72</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>名字</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="student_sname">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>72</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>性别</string>
     </property>
    </widget>
    <widget class="QComboBox" name="student_ssex">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>男</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>女</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>160</y>
       <width>72</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>所在系</string>
     </property>
    </widget>
    <widget class="QComboBox" name="student_sdept">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>160</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>计算机</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>新闻</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>金融</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>数学</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>外语</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>260</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>联系电话</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="student_stel">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>260</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="student_save_button">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>410</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>页</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
